package.json

```diff
@@ -11,20 +11,20 @@
 		"url": "https://github.com/Tradeshift/eslint-config-tradeshift/issues"
 	},
 	"dependencies": {
-		"babel-eslint": "^8.2.3",
+		"babel-eslint": "8.2.5",
 		"eslint-config-prettier": "2.9.0",
-		"eslint-config-standard": "11.0.0",
-		"eslint-plugin-import": "2.12.0",
+		"eslint-config-standard": "12.0.0-alpha.0",
+		"eslint-plugin-import": "2.13.0",
 		"eslint-plugin-node": "6.0.1",
 		"eslint-plugin-promise": "3.8.0",
 		"eslint-plugin-standard": "3.1.0"
 	},
 	"peerDependencies": {
-		"eslint": ">=3"
+		"eslint": "5.1.0"
```

Comment:
I really don't like the fixed versions.

It's okayish for the embedded ones, but for a `peerDependency` I think it's a bit extreme/weird.

Reply:
but that means that we will have to deal with different ESLint versions using our config

that could cause unexpected errors all the way

Reply:
Hopefully not, we could fix it to a major and then assume no breaking changes according to the spec.

If it causes issues: Allowing projects to specify their own version will help find the issues gradually, whereas freezing everything and then dealing with bumping/breaking everything from here sounds painful.

Reply:
I can push `>=5.1.0` if that would make you feel better, but I think normally we should require these things to be kind of stable, in case there is a mismatch of versions

Reply:
As a library I think we should be as liberal as possible with externals dependencies. Then it's up to the consumer to lock it down if they want to, enforcing consistent behaviour/versions on a per-project basis.

Reply:
Use a `~` or `^` range and I'm happy :)